test/models/language_skill_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class LanguageSkillTest < ActiveSupport::TestCase
+  test 'language skill is valid' do
+    german = LanguageSkill.create!(language: 'DE', level: 'good')
+    assert_equal 1, LanguageSkill.count
+    assert german.valid?
```

Comment:
Note that `test/models/factory_test.rb` already makes sure all factories are valid, so we don't really need this test :)